app/services/git/syncs_tracks.rb

```diff
@@ -1,41 +1,24 @@
 class Git::SyncsTracks
```

Comment:
I think this class is almost redundant now. Same for `FetchesTracks` possibly, and the `Git::StateDb`. Your solution is much better.

Reply:
I think we should still keep this in order to have an easy way to sync all tracks, just as I have described for `FetchesTracks`. What do you think?